plugins/org.eclipse.xtext.ui.shared/plugin.properties

```diff
@@ -2,7 +2,7 @@ OpenModelElementCommand.name=Open Model Element
 OpenModelElementCommand.description=Open the declaration of an Xtext element
 XtextBuilder.name=Xtext Project Builder
 XtextNature.name=Xtext Project Nature
-XtextNatureAddAction.label=Add Xtext Nature
-XtextNatureRemoveAction.label=Remove Xtext Nature
+XtextNatureAddAction.label=Convert to Xtext project
+XtextNatureRemoveAction.label=Unconfigure Xtext
```

Comment:
Maybe makes sense. It is unlikely that someone disables Xtext once it was configured. But if someone does want to do this, how should it be done? I think it is too prominent to have this option always in the "Configure" menu. How about having this in the project preference page? If so, shall I open a separate bug?